From 285fa89b030ac9f525d0283d10bbb7a6d4b2b6b2 Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gmail.com>
Date: Wed, 20 Jul 2011 05:31:55 -0400
Subject: [PATCH] Make the clutter and mash dependency non-automagic (#654954)

Ensure that there is a way of building gnome-color-manager without
clutter and 3D support on a system that happens to have clutter-gtk
and mash installed. (Gnome bug #654954)
---
 configure.ac |    8 +++++++-
 1 files changed, 7 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index ee1760e..02ad961 100644
--- a/configure.ac
+++ b/configure.ac
@@ -97,7 +97,13 @@ PKG_CHECK_MODULES(LCMS, lcms2 >= 2.2)
 PKG_CHECK_MODULES(X11, x11)
 PKG_CHECK_MODULES(COLORD, colord >= 0.1.9)
 
-PKG_CHECK_MODULES(CLUTTER, clutter-gtk-1.0 mash-0.1, has_clutter=yes, has_clutter=no)
+dnl **** Check for clutter ****
+AC_ARG_ENABLE(clutter, AS_HELP_STRING([--enable-clutter],[Enable clutter for 3D support]), enable_clutter=$enableval, enable_clutter=yes)
+if test x$enable_clutter != xno; then
+	PKG_CHECK_MODULES(CLUTTER, clutter-gtk-1.0 mash-0.1, has_clutter=yes, has_clutter=no)
+else
+	has_clutter=no
+fi
 AM_CONDITIONAL(HAVE_CLUTTER, test x$has_clutter = xyes)
 if test x$has_clutter = xyes; then
 	AC_DEFINE(HAVE_CLUTTER,1,[has clutter support])
-- 
1.7.6

